clean_all.cmd

```diff
@@ -1,18 +1,34 @@
+@ECHO OFF
 
-del .\GKCommon.*
-del .\GEDKeeper2.exe.*
-del .\GKUpdater.*
-del .\GKTray.*
-del .\*.dll
-del .\*.pdb
-del .\*.mdb
-del .\*.xml
-del .\*.log
-del .\plugins\*.* /q
-del .\deploy\*.zip /q
+@ECHO/
+@ECHO *****************
+@ECHO ** Cleaning up **
+@ECHO *****************
+@ECHO/
 
-del .\GEDKeeper3.Gtk2.exe.*
-del .\GEDKeeper3.Gtk3.exe.*
-del .\GEDKeeper3.Mac.exe.*
-del .\GEDKeeper3.WinForms.exe.*
-del .\GEDKeeper3.Wpf.exe.*
+if exist .\LICENSE GOTO Clean_Up
+@ECHO/
+@ECHO ERROR: You are not in the correct directory to run this.
+pause
+exit /b 1
+
+:Clean_Up
+    if exist .\GKCommon.*                   del /q .\GKCommon.*
+    if exist .\GEDKeeper2.exe.*             del /q .\GEDKeeper2.exe.*
+    if exist .\GKUpdater.*                  del /q .\GKUpdater.*
+    if exist .\GKTray.*                     del /q .\GKTray.*
+    if exist .\*.dll                        del /q .\*.dll
+    if exist .\*.pdb                        del /q .\*.pdb
+    if exist .\*.mdb                        del /q .\*.mdb
+    if exist .\*.xml                        del /q .\*.xml
+    if exist .\*.log                        del /q .\*.log
+    if exist .\plugins\*.*                  del /q .\plugins\*.*
+    if exist .\deploy\*.zip                 del /q .\deploy\*.zip
+
+    if exist .\GEDKeeper3.Gtk2.exe.*        del /q .\GEDKeeper3.Gtk2.exe.*
+    if exist .\GEDKeeper3.Gtk3.exe.*        del /q .\GEDKeeper3.Gtk3.exe.*
+    if exist .\GEDKeeper3.Mac.exe.*         del /q .\GEDKeeper3.Mac.exe.*
+    if exist .\GEDKeeper3.WinForms.exe.*    del /q .\GEDKeeper3.WinForms.exe.*
+    if exist .\GEDKeeper3.Wpf.exe.*         del /q .\GEDKeeper3.Wpf.exe.*

```

Comment:
Check for tabs (only spaces are allowed).

Reply:
Checked for tabs, and none were found.

Reply:
```sh
$ grep -e $'\t' clean_all.cmd | wc -w
0
```

;-)

Reply:
For an accurate file comparison, I deleted the spaces by 4 in a row and somewhere apparently incorrectly evaluated the result - it seemed that there were tabs. Apparently wrong